suite-common/test-utils/src/extraDependenciesMock.ts

```diff
@@ -75,6 +75,10 @@ export const extraDependenciesMock: ExtraDependencies = {
         selectBitcoinAmountUnit: mockSelector('selectBitcoinAmountUnit', PROTO.AmountUnit.BITCOIN),
         selectAreSatsAmountUnit: mockSelector('selectAreSatsAmountUnit', false),
         selectEnabledNetworks: mockSelector('selectEnabledNetworks', ['btc', 'test']),
+        selectTokenDefinitionsEnabledNetworks: mockSelector(
+            'selectTokenDefinitonsEnabledNetworks',
+            ['eth'],
```

Comment:
Maybe you can make selector, that fetches all accounts, take those imported, get their symbols and check with `isCoinWithTokens` and return all token enabled network symbols. 

This way even before change how token definitions are fetched, we would have only one place where to set which definitions are enabled. We would just update `isCoinWithTokens` with polygon or whatever